### ISSUE TITLE
Question: Possible to retrieve GPU information like model, sw version, etc.?

### ISSUE THREAD

**Issue by DeadEnded** [mark9064/pyamdgpuinfo#5]
Hello again!
I'm back playing with pyamdgpuinfo - I've made progress - but was now wondering if it was possible to retrieve any of the GPU information like model, manufacturer, software version, etc.?  I expect the answer is no, but I thought it was worth asking.

Just a follow up for a few month ago - I have learned a lot about the Home Assistant development and how everything works.  I actually have a working draft of a custom component that uses pyamdgpuinfo!  I am trying to clean up some parts before I consider it a final first draft!  Just wanted to say than you for your help!

Cheers!
DeadEnd

**Comment by mark9064** [comment]
Hey there,
What other information were you thinking of specifically? As for specifics, the model name is already available, and the manufacturer will always be AMD :P

Super cool to hear about your home assistant, I hope it works out well :)

**Comment by DeadEnded** [comment]
okay... I failed to figure out how to retrieve it then.
I assumed it was in GPUInfo, but I think last I tried I was just getting an object and didn't know how to get data out of it.
Could you be so kind as to fill in my knowledge gap?
How do you retrieve this information correctly?

I expect the model would be enough... I mean software/firmware version is neat and all but completely not necessary... lol.
Likely whatever is already available is enough for what I am looking at... just failed to find it 🤷 .

Thanks again!

**Comment by mark9064** [comment]
The GPUInfo class is documented [here](https://github.com/mark9064/pyamdgpuinfo/blob/master/pyamdgpuinfo/pyamdgpuinfo.pyx#L231). You should also be able to do `help(pyamdgpuinfo.GPUInfo)` in python repl or your IDE should show it

**Comment by DeadEnded** [comment]
Yes, I saw the information in the code... but I am not sure how to execute it correctly.
For example, I tried the following:
`pyamdgpuinfo.GPUInfo("/dev/dri/card0",0)`
`print(pyamdgpuinfo.GPUInfo("/dev/dri/card0",0))`
 and get `<pyamdgpuinfo.pyamdgpuinfo.GPUInfo object at 0x7f7dce29b740>` in response.
.....
Okay - I kept trying random things and finally think I figured it out.
I tried `dir(pyamdgpuinfo.GPUInfo("/dev/dri/card0",0))` to get a list of all the attributes, etc.
Then I finally found the very simple `pyamdgpuinfo.GPUInfo("/dev/dri/card0",0).name`.
```
>>> pyamdgpuinfo.GPUInfo("/dev/dri/card0",0).name
'AMD Radeon(TM) Vega 11 Graphics'
```
Okay - I should be able to figure it out from here...
... much easier... `pyamdgpuinfo.get_gpu(0).name`... no path needed
Thanks for the response... just had to work through my knowledge gap!

Cheers!
DeadEnd
